common/file.yaml

```diff
@@ -0,0 +1,21 @@
+groups:
+- data_type_def: HDMFFile
+  data_type_inc: Container
+  doc: A base Container type for a logical file
+  groups:
+  - name: .ontologies
```

Comment:
I believe Lawrence said something about MatNWB not being able to handle group names that start with a dot, and he suggested underscore instead.